900.version-fixes/g.yaml

```diff
@@ -144,95 +141,46 @@
 - { name: gnome-applets,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-authenticator,         verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-backgrounds,           verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-bluetooth,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-books,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-boxes,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-builder,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-calculator,            verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-calculator,            verge: "5", verlt: "7",                             sink: true } # old version scheme from gcalctool
-- { name: gnome-calendar,              verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-characters,            verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-chess,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-clocks,                verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-color-manager,         verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-contacts,              verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-contacts,              ver: "3.32.2",                ruleset: void,        incorrect: true } # 3.32 actually
 - { name: gnome-contacts,                                            ruleset: void,        untrusted: true } # accused of fake 3.32.2
-- { name: gnome-control-center,        verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-desktop,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-devel-docs,            verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-disk-utility,          verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-documents,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-flashback,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-font-viewer,           verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-games,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-games-app,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-getting-started-docs,  verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
```

Comment:
I included this one because the project is now archived (i.e. dead). See https://gitlab.gnome.org/Archive/gnome-getting-started-docs

I'd be happy to keep the rule in any case—whatever you prefer.